### DISCUSSION TITLE
Hiding all sprites

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7140] [Ideas]
When my students have to test a specific part of a large project, with many sprites, it is annoying to them having to go to every single sprite that is visible on the Stage and hide it, so they do not interfere with the part you are testing.

It would be really great having, e.g., an option in a contextual menu that when you right-click the Stage says "Hide all sprites".

**Comment by CubesterYT** [comment]
I like this idea!

**Comment by Samq64** [comment]
> It would be really great having, e.g., an option in a contextual menu that when you right-click the Stage says "Hide all sprites".

That could be part of #7077.

**Reply by WorldLanguages** [reply]
Yep.

**Comment by DNin01** [comment]
This could also be extended to folders, if you have the addon enabled - what if you could right-click on sprite folders for an option to hide or show all sprites in the folder?
